backend/de.metas.business/src/test/java/de/metas/business/BusinessTestHelper.java

```diff
@@ -209,6 +209,7 @@ public I_M_Product createProduct(
 		product.setC_UOM_ID(UomId.toRepoId(uomId));
 		product.setProductType(ProductType.Item.getCode());
 		product.setIsStocked(true);
+		product.setM_Product_Category_ID(ProductCategoryId.toRepoId(createProductCategory("Standard", null)));
```

Comment:
so u will create a new product category each time you will create a product?

Reply:
adjusted